NuWeather/plugin.py

```diff
@@ -377,7 +407,7 @@ def _darksky_fetcher(self, location):
         # N.B. Dark Sky docs tell to not expect any values to exist except the timestamp attached to the response
         return {
             'location': display_name,
-            'poweredby': 'Dark\xa0Sky+' + self._geocode.backend,
+            'poweredby': 'Dark\xa0Sky+' + self.geoback.title(),
```

Comment:
Instead of this maybe we can add the backend name to _geocode()'s return value? (If it's missing from previously cached values, assume it's Nominatim)

Reply:
Addded a check for osm_id in geocodeid, should accomplish the same thing